web/pgadmin/tools/sqleditor/static/js/components/sections/MainToolBar.jsx

```diff
@@ -388,6 +396,12 @@ export function MainToolBar({containerRef, onFilterClick, onManageMacros}) {
         });}
       }
     },
+    {
+      shortcut: queryToolPref.execute_script,
```

Comment:
why do we need this? Its already taken care with other shortcuts.

Reply:
Are we migrating the old shorcut preference set by the user?

Reply:
Yes